array.ts

```diff
@@ -345,6 +346,23 @@ export const unzip = <A, B>(
   return [fa, fb];
 };
 
+/**
+ * Returns a new array containing elements of `as` sorted in ascending order
+ * according to the sort order defined by `O`.
+ *
+ * @example
+ * import { ordNumber } from "https://deno.land/x/fun/ord.ts";
+ * import { sort } from 'https://deno.land/x/fun/array.ts'
```

Comment:
Let's use local paths for examples, i.e. `import { ordNumber } from "./ord.ts";`.